tests/parm/ufs.configure.cpld_s2sa_p8.IN

```diff
@@ -0,0 +1,135 @@
+#############################################
```

Comment:
We need to think about how to name the config files. Currently we have ``noaero_nowave`` etc but I think I like this way of naming it w/ the application (``ufs.confgure.cpld_s2sa``) better. 

We won't use the ``_p8`` label however, since these config files are not specific to a certain physics configuration (and ``p8`` is going away anyway.